pkg/collector/corechecks/oracle-dbm/oracle.go

```diff
@@ -223,3 +253,11 @@ func (c *Check) GetObfuscatedStatement(o *obfuscate.Obfuscator, statement string
 func (c *Check) getFullPDBName(pdb string) string {
 	return fmt.Sprintf("%s.%s", c.cdbName, pdb)
 }
+
+func appendPDBTag(tags []string, pdb sql.NullString) []string {
+	if pdb.Valid {
+		return append(tags, "pdb:"+pdb.String)
+	} else {
+		return tags
```

Comment:
This is fine but the preferred way to do this is to do something like:

```suggestion
	if !pdb.Valid {
		return tags
	} 
	return append(tags, "pdb:"+pdb.String)
```